code/frontend/src/app/ui/input/input.component.scss

```
@@ -18,7 +18,7 @@
 
 .input-field {
   @include glass-input;
-  height: 38px;
+  height: var(--input-height);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)